src/asar-utils.ts

```diff
@@ -31,3 +58,153 @@ export const generateAsarIntegrity = (asarPath: string) => {
       .digest('hex'),
   };
 };
+
+function toRelativePath(file: string): string {
+  return file.replace(/^\//, '');
+}
+
+function isDirectory(a: string, file: string): boolean {
+  return Boolean('files' in asar.statFile(a, file));
+}
+
+function checkUnique(archive: string, file: string, allowList?: string): void {
+  if (allowList === undefined || !minimatch(file, allowList, { matchBase: true })) {
+    throw new Error(
+      `Detected unique file "${file}" in "${archive}" not covered by ` +
+        `allowList rule: "${allowList}"`,
+    );
+  }
+}
+
+export const fuseASARs = async ({
+  x64,
+  arm64,
+  output,
+  lipo = 'lipo',
```

Comment:
Is this the path / name of the `lipo` binary? If so let's go with `lipoPath`, more indicative

Reply:
Also why is this an option, it isn't a user-facing option and afaics we don't do anything similar elsewhere

Reply:
Agreed. Let's drop this.